addons/analytic/models/analytic_mixin.py

```diff
@@ -48,17 +48,21 @@ def _compute_analytic_distribution(self):
         pass
 
     def _search_analytic_distribution(self, operator, value):
-        if operator not in ['=', '!=', 'ilike', 'not ilike'] or not isinstance(value, (str, bool)):
-            raise UserError(_('Operation not supported'))
-        operator_name_search = '=' if operator in ('=', '!=') else 'ilike'
-        account_ids = list(self.env['account.analytic.account']._name_search(name=value, operator=operator_name_search))
+        if operator == 'in' and isinstance(value, (tuple, list)):
+            account_ids = value
+            operator_inselect = 'inselect'
+        else:
+            if operator not in ['=', '!=', 'ilike', 'not ilike'] or not isinstance(value, (str, bool)):
+                raise UserError(_('Operation not supported'))
```

Comment:
```suggestion
                raise UserError(_("Operation not supported"))
```

Reply:
This is also from 17.0, and is there with single quotes, so maybe better to keep it that way for the diff?